framework/web/AssetManager.php

```diff
@@ -260,6 +261,11 @@ protected function loadBundle($name, $config = [], $publish = true)
         if (!isset($config['class'])) {
             $config['class'] = $name;
         }
+        
+        if (isset($this->bundles['*'])) {
+            $config = array_merge($config, $this->bundles['*']);
```

Comment:
Should it be the other way around? i.e. what's specified in `$this->bundles['*']` is default values and what's passed to the method are more specific values.